package.json

```diff
@@ -1,39 +1,38 @@
 {
-  "name": "client",
-  "version": "0.1.0",
-  "private": true,
-  "dependencies": {
-    "@testing-library/jest-dom": "^4.2.4",
-    "@testing-library/react": "^9.5.0",
-    "@testing-library/user-event": "^7.2.1",
-    "apollo-boost": "^0.4.7",
-    "mobx": "^5.15.4",
-    "react": "^16.13.1",
-    "react-apollo": "^3.1.4",
-    "react-dom": "^16.13.1",
-    "react-router-dom": "^5.1.2",
-    "react-scripts": "3.4.1",
-    "redux": "^4.0.5"
-  },
-  "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
-    "eject": "react-scripts eject"
-  },
-  "eslintConfig": {
-    "extends": "react-app"
-  },
-  "browserslist": {
-    "production": [
-      ">0.2%",
-      "not dead",
-      "not op_mini all"
-    ],
-    "development": [
-      "last 1 chrome version",
-      "last 1 firefox version",
-      "last 1 safari version"
-    ]
-  }
+	"name": "client",
+	"version": "0.1.0",
+	"private": true,
+	"dependencies": {
+		"@testing-library/jest-dom": "^4.2.4",
+		"@testing-library/react": "^9.5.0",
+		"@testing-library/user-event": "^7.2.1",
```

Comment:
Move these three items to **devDependencies**.
What can I keep in devDependencies? Any package/dependency that doesn't need to run in the production server. In this case, testing ends in the CI

Reply:
You have a very simple but serious issue with your editor. you can have a call with Chialuka to fix this 

or add **.editorconfig** in your root directory. https://editorconfig.org/

Reply:
I have done as you directed sir.